### DISCUSSION TITLE
[Build] Validate Package ReferenceAPI Failed

### DISCUSSION THREAD

**Discussion by workgroupengineering** [unoplatform/uno#8138] [Q&A]
### Current behavior

When compiling PR #7830, API validation fails because I have added private [ctor](https://github.com/unoplatform/uno/blob/5bbb7b44d92feefc698f5a25f2752ef8d8f033f4/src/Uno.UWP/Graphics/Imaging/BitmapEncoder.skia.cs#L26-L31) in BitmapEncoder.

I have added ignore to [PackageDiffIgnore.xml](https://github.com/unoplatform/uno/blob/5bbb7b44d92feefc698f5a25f2752ef8d8f033f4/build/PackageDiffIgnore.xml#L8666-L8671)  but it fails because ValidatePackageReferenceAPI does not use it.

https://github.com/unoplatform/uno/blob/58809ef8591ca9582a6f89287b96159e4fad97cb/build/Uno.UI.Build.csproj#L440-L457

My questions are:
- Why double check the API(see above)?
- Can ValidatePackageReferenceAPI Target be removed?

### Environment

Other

### NuGet package version(s)

master 58809ef8591ca9582a6f89287b96159e4fad97cb

### Affected platforms

Build tasks

### IDE

Visual Studio 2022

### IDE version

17.1.0 Preview 1.1

**Comment by jeromelaban** [comment]
No, the `ValidatePackageReferenceAPI` target cannot be removed as it ensure that the reference API matches actual platform implementations. Failing to align the targets will cause either runtime issues, or linker issues. If you removed the target in your PR, restore it and show the error in the PR, we can discuss it there.
